defaults/main.yml

```
@@ -1,62 +1,55 @@
 ---
 
-# Controls whether miniconda will be installed and/or updated
-miniconda_install: true
-miniconda_update: true
-
 # Installers are not released for every version of conda, you can use this to
-# `conda install conda={{ miniconda_version }}` a specific version of conda. If set to `latest` then the role will
+# `conda install conda={{ miniforge_version }}` a specific version of conda. If set to `latest` then the role will
 # perform a `conda update conda` instead.
-miniconda_version: latest
+miniforge_version: latest
 
 # If you wish to install e.g. a conda-forge version of conda you can do so by setting channels. This causes
 # `--override-channels --channel <channel> [--channel <channel> ...]` to be specified on the call to
 # `conda install conda...` or `conda update conda`.
-miniconda_channels: []
+miniforge_channels: []
```

```suggestion
conda_channels: []
```

and change to default to `miniconda_channels` if set.